src/app/category/controller.js

```diff
@@ -15,11 +15,30 @@ const createCategory = async (req, res) => {
 };
 
 const getCategories = async (req, res) => {
+  const userId = (req.decoded && req.decoded.id) || '';
```

Comment:
Please add `/* istanbul ignore next */` here since we're not testing for it yet.

Reply:
Ok